### ISSUE TITLE
Option to remotely update and change the miner?

### ISSUE THREAD

**Issue by vladscorpius** [UnamSanctam/SilentCryptoMiner#1823]
Maybe I'm stupid but I don't see this as an option on the builder or the miner on the newest version.

Is this something I need the web panel for, or is there a setting i'm missing to where if something goes wrong I can change it without having physical access to the computer?

**Comment by vladscorpius** [comment]
Also, if I go to the settings menu of the miner after having already built it, can I change the settings without having to create a new miner, does it just automatically update?

**Comment by UnamSanctam** [comment]
To change the miner settings remotely you can use the "Remote Configuration" feature or the web panel. For remote updating (running the uninstaller and then installing a completely new miner) you would need something like a RAT on that computer and do it through that.

**Comment by vladscorpius** [comment]
thank you! one more question, when I have the 64-bit file for the miner on my desktop, and send it to someone, say via mega file sharing, does it also go on their desktop or does it automatically hide itself in their computer as soon as they download it?

**Comment by UnamSanctam** [comment]
It goes where they save it (like the Desktop) and then they'll have to start/run the file.. You can't really make a file/program do anything automatically in Windows just by getting downloaded.

**Comment by vladscorpius** [comment]
So if I were to physically download this on someones computer, would putting it on mega work to download it to their computer, and then I would have to go in and actually double click on it to start it?

**Comment by UnamSanctam** [comment]
Yes like any other program you first download it and then double-click it. It's not really (usually) possible to do it any other way for security reasons.
